data/common_voice.py

```diff
@@ -82,7 +85,7 @@ def main():
         tar.extractall(target_unpacked_dir)
         tar.close()
 
-    folder_path = os.path.join(target_unpacked_dir, VERSION + '/en/')  # TODO expose the language flag
+    folder_path = os.path.join(target_unpacked_dir, VERSION + '/{}/'.format(language_dir))  # TODO expose the language flag
```

Comment:
remove comment I guess since it's no longer needed.

Reply:
Done!